<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="2,2" columnstretch="2,3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
        <item>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0">
          <item row="6" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QSlider" name="lat_min_slider">
              <property name="minimum">
               <number>-90</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>-90</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>lon max</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSlider" name="lon_max_slider">
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="value">
               <number>180</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>lon min</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>lat max</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>lat min</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <widget class="QSlider" name="lon_min_slider">
              <property name="minimum">
               <number>-180</number>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="value">
               <number>-180</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="lat_max_slider">
              <property name="minimum">
               <number>-90</number>
              </property>
              <property name="maximum">
               <number>90</number>
              </property>
              <property name="value">
               <number>90</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="Vp_button">
              <property name="text">
               <string>Vp</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="Vs_button">
              <property name="text">
               <string>Vs</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="8" column="0">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="2">
             <widget class="QCheckBox" name="hotspots_box">
              <property name="text">
               <string>hotspots</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="plate_boundaries_box">
              <property name="text">
               <string>plate boundaries</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="coastlines_box">
              <property name="text">
               <string>coastlines</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="cbar_max">
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>colorbar max (%)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QSlider" name="depth_slider">
            <property name="minimum">
             <number>25</number>
            </property>
            <property name="maximum">
             <number>2891</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>parameter</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>depth (km)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10,1">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>S40RTS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>S362ANI</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GAP_P4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MITP08</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="Qt4MplCanvas" name="global_map"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="submit_button">
            <property name="text">
             <string>submit</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="Qt4MplCanvas" name="local_map"/>
        </item>
        <item>
         <widget class="QPushButton" name="cross_section_button">
          <property name="text">
           <string>calculate cross section</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="Qt4MplCanvas" name="cross_section"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Qt4MplCanvas</class>
   <extends>QGraphicsView</extends>
   <header>instaseis.gui.qt4mplcanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
